src/js/containers/app/main-header.js

```diff
@@ -16,40 +16,53 @@ const MainHeader = () => (
     <a className='logo' href='./' role='logo'>
       <img src='img/logo.png' />
     </a>
-    <a
-      className='btn-link'
-      target='_blank'
-      href='https://frontendmeetup.typeform.com/to/vNNQyd'
-      title='Quero apresentar'>
-        Quero palestrar
-    </a>
-    <a
-      className='btn-link'
-      target='_blank'
-      href='https://frontendmeetup.typeform.com/to/XuXAD5'
-      title='Quero sediar'>
-        Quero sediar
-    </a>
-    <a
-      className='btn-link'
-      href='#manifest'
-      title='Manifesto Campinas front-end'>
-        Manifesto
-    </a>
-    <a
-      className='btn-link'
-      href='#conduct'
-      title='Código de conduta'>
-        Código de conduta
-    </a>
-
-    <a
-      className='btn-link'
-      target='_blank'
-      href='http://frontendcampinas.herokuapp.com/'
-      title='Slack'>
-        Slack
-    </a>
+    <nav className='menu' role='navigation'>
+      <ul>
```

Comment:
@gabriel-ribeiro-ir se preeeeeeeeeeeecisa mesmo eu não sei. Não tem nenhum artigo que fale que precisa e ponto. Porém todos os exemplos [Mozilla Developers Network](http://w3c.github.io/html/sections.html#the-nav-element), [W3C](http://w3c.github.io/html/sections.html#the-nav-element) e outros, são mostrados navs com lista. 

A única literatura contra que achei foi um [artigo de 2013 do CSS Tricks](https://css-tricks.com/navigation-in-lists-to-be-or-not-to-be/) que levanta o questionamento e alguns pontos mas finaliza com [este outro artigo](https://css-tricks.com/wrapup-of-navigation-in-lists/) declarando meio que um empate mas tendo, pessoalmente, pelo uso de lista também.

Mas como parece não ter nenhum dado realmente concreto de porquê usar listas, a gente deixa esse PR pra lá e seguir com um projeto disruptivão. 😂